nemoguardrails/library/zscaler_aiguard/actions.py

```
@@ -0,0 +1,234 @@
+
+        return {
+            "action": action_val,
+            "severity": severity,
+            "policy_name": policy_name,
+            "transaction_id": transaction_id,
+            "detectors": detectors,
+            "blocking_detectors": blocking_detectors,
+            "message": message,
+        }
+
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)